serverless.yml

```diff
@@ -36,7 +36,7 @@ functions:
   newrelic-log-ingestion:
     description: Send log data from CloudWatch Logs to New Relic Infrastructure (Cloud Integrations) and New Relic Logging.
     handler: src/function.lambda_handler
-    name: newrelic-log-ingestion
+    name: newrelic-log-ingestion-akhilv1
```

Comment:
why this name change?

Reply:
Thanks for catching that. This was an unintended change made during my testing. I will revert the function name to its original value and push the correction shortly